### DISCUSSION TITLE
Override certain config.theme.features (namely navigation.expand)

### DISCUSSION THREAD

**Discussion by bobek** [squidfunk/mkdocs-material#4902] [Q&A] [answered]
First of all, thank you so much for `mkdocs-material`.

We have enabled `config.theme.features.nagivation.expand` as it makes easier for people to find nested pages. But I would like to disable it for certain sub-trees as expansion makes them pretty messy ( like https://docs.ipfabric.io/main/releases/ ).

It seems that config is evaluated only once. Any chance to make it part of meta (fully aware that it may not make any sense for the broader audience).

Thank you.

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. This is currently not possible, but investigation into whether certain features can be disabled for certain subtrees or pages is planned. It's particularly challenging to support for features like instant loading, but I think it should be possible, at least for the majority of features!
